app/src/main/java/com/akshay/newsapp/news/model/NewsArticles.kt

```diff
@@ -14,6 +16,7 @@ import com.google.gson.annotations.SerializedName
  * Warning: Ideally network and database model should be different.
  * They are same for this sample app.
  */
+@Parcelize
```

Comment:
We can remove the `Parcelize` once we use the ID to communicate between activities